README.md

```diff
@@ -553,29 +591,39 @@ Now that the nodes have been initialized, we rotate the secrets files to overwri
 ```
   $docker secret create mldb_admin_password_v2 mldb_admin_password_v2.txt
 ```
+- Create mldb_wallet_password_v2.txt and use the following command to add a new docker secret for the wallet password:
+```
+  $docker secret create mldb_wallet_password_v2 mldb_wallet_password_v2.txt
+```
 6. Use the following commands to rotate the docker secrets for all the docker services created above using docker stack:
```

Comment:
Use the following commands to rotate the Docker secrets for all the Docker services created above using Docker stack:

(capitalize Docker)

Reply:
Fixed.